Setup_bam_logging.sh fails silently on re-install

swgkg opened the pull request:
Please review and merge

This PR contains the fixes for 
PAAS-140- Setup_bam_logging.sh fails silently on re-install

imesh commented:
Gayan: This pull request has become obsolete now as we have moved master branch to ppaas 4.1.0 development. Can you please confirm whether we have already applied this fix to Stratos 4.1.0?
Thanks

swgkg commented:
Thanks Imesh for checking this.
Seems this is not available in the code. This is a small fix. Added -o option for unzip .This is to  overwrite files without prompting when unzip the distribution. As we have moved the master branch to ppaas 4.1.0 , we can add this change manually to ppaas 4.1.0. So we can close this PR.
